bundles/org.openhab.binding.echonetlite/src/main/java/org/openhab/binding/echonetlite/internal/EchonetLiteBridgeHandler.java

```diff
@@ -0,0 +1,399 @@
+    private final Clock clock = Clock.systemUTC();
+
+    @Nullable
+    private EchonetChannel echonetChannel;
+
+    @Nullable
+    private InstanceKey managementControllerKey;
+
+    @Nullable
+    private InstanceKey discoveryKey;
+
+    public EchonetLiteBridgeHandler(Bridge bridge) {
+        super(bridge);
+    }
+
+    public void start() throws IOException {
+
+        if (null == discoveryKey) {
+            logger.error("Discovery key is null, unable to start");
+            return;
+        }
+
+        logger.info("Binding echonet channel");
+        echonetChannel = new EchonetChannel(requireNonNull(discoveryKey).address);
+        logger.info("Starting networking thread");
+
+        networkingThread.setName("Echonet Networking");
+        networkingThread.setDaemon(true);
+        networkingThread.start();
+    }
+
+    public void newDevice(InstanceKey instanceKey, long pollIntervalMs, long retryTimeoutMs,
+            final EchonetDeviceListener echonetDeviceListener) {
+        requests.add(new NewDeviceMessage(instanceKey, pollIntervalMs, retryTimeoutMs, echonetDeviceListener));
+    }
+
+    private void newDeviceInternal(final NewDeviceMessage message) {
+        final EchonetObject echonetObject = devicesByKey.get(message.instanceKey);
+        if (null != echonetObject) {
+            if (echonetObject instanceof EchonetDevice) {
+                logger.debug("Update item: {} already discovered", message.instanceKey);
+                EchonetDevice device = (EchonetDevice) echonetObject;
+                device.setTimeouts(message.pollIntervalMs, message.retryTimeoutMs);
+                device.setListener(message.echonetDeviceListener);
+            } else {
+                logger.debug("Item: {} already discovered, but was not a device", message.instanceKey);
+            }
+        } else {
+            logger.debug("New Device: {}", message.instanceKey);
+            final EchonetDevice device = new EchonetDevice(message.instanceKey, message.echonetDeviceListener);
+            device.setTimeouts(message.pollIntervalMs, message.retryTimeoutMs);
+            devicesByKey.put(message.instanceKey, device);
+        }
+    }
+
+    public void refreshDevice(final InstanceKey instanceKey, final String channelId) {
+        requests.add(new RefreshMessage(instanceKey, channelId));
+    }
+
+    private void refreshDeviceInternal(final RefreshMessage refreshMessage, long nowMs) {
+        final EchonetObject item = devicesByKey.get(refreshMessage.instanceKey);
+        if (null != item) {
+            item.refresh(refreshMessage.channelId);
+        }
+    }
+
+    public void removeDevice(final InstanceKey instanceKey) {
+        requests.add(new RemoveDevice(instanceKey));
+    }
+
+    private void removeDeviceInternal(final RemoveDevice removeDevice) {
+        final EchonetObject remove = devicesByKey.remove(removeDevice.instanceKey);
+
+        logger.info("Removing device: {}, {}", removeDevice.instanceKey, remove);
```

Comment:
Please reduce log level to DEBUG or TRACE. See https://next.openhab.org/docs/developer/guidelines.html#f-logging